spec/lib/samvera/nesting_indexer/adapters/abstract_adapter_spec.rb

```diff
@@ -43,7 +43,13 @@ module Adapters
         end
 
         describe '.write_document_attributes_to_index_layer' do
-          subject { described_class.write_document_attributes_to_index_layer(id: 1, parent_ids: 2, ancestors: 3, pathnames: 4) }
+          subject { described_class.write_document_attributes_to_index_layer(id: 1, parent_ids: 2, ancestors: 3, pathnames: 4, deepest_nested_depth: 5) }
+          it 'requires implementation (see documentation)' do
+            expect { subject }.to raise_error(NotImplementedError)
```

Comment:
Would it be better to write this as a pending test?  That way it would show up in the rspec log as pending giving a reminder to come back to it.

Reply:
I will rephrase this. The methods are part of an abstract adapter; The abstract adapter methods define method signature but not implementation. The class that extends this adapter would then need to implement.